Lib/Rabbit4000/BIOSLIB/StdBios.c

```diff
@@ -1874,6 +1874,13 @@ ioi   ld    (STACKSEG), a
 #endif
 //end serial flash fast RAM copy
 
+#if _BOARD_TYPE_ == RCM4110
+; RCM4110 with 55ns flash requires early output enable on MTCR when running
+; with 0 wait states, so configure that before setting MB0CR.
+      ld    a, 0x0C
+ioi   ld    (MTCR), a
+#endif
+

```

Comment:
This change enables "early /OE" on the RCM4110 before reaching code that configures `MB0CR` for 0 wait states, and therefore will access the RCM4110's 55ns flash at full speed.